src/display/font_loader.js

```diff
@@ -216,7 +218,8 @@ if (typeof PDFJSDev === 'undefined' || !PDFJSDev.test('MOZCENTRAL')) {
 
       var i, ii;
 
-      var canvas = document.createElement('canvas');
```

Comment:
revert the change here -- it is used for hack to determine if font is loaded, you can add a comment that CanvasRactory is not needed here